src/xenia/vfs/devices/stfs_xbox.h

```diff
@@ -10,12 +10,37 @@
 #ifndef XENIA_VFS_DEVICES_STFS_XBOX_H_
 #define XENIA_VFS_DEVICES_STFS_XBOX_H_
 
+#include "xenia/xbox.h"
 #include "xenia/base/string_util.h"
 #include "xenia/kernel/util/xex2_info.h"
 
+#if XE_PLATFORM_WIN32
+#include "xenia/base/platform_win.h"
+#define timegm _mkgmtime
```

Comment:
It'd be less intrusive to use:
```cpp
#if XE_PLATFORM_WIN32
time_t timet = _mkgmtime(&tm);
#else
time_t timet = timegm(&tm);
#endif
```
at the call location instead of redefining a platform function name to something else (or even a `src/xenia/base` wrapper function or at least a `XE_` definition for better reusability, but if it's needed in only one place that's not necessary). The previously used approach was relatively fine in `.cc` files (though still pretty dirty), but in headers it's dangerous as it may affect other headers included after this one.